wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -431,6 +431,8 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
   }
 
   Stack buildOpenedSheet(BuildContext context, PurchaseItemViewModel viewModel) {
+    final walletStore = GetIt.I.get<WalletsStore>();
```

Comment:
Whats the reason for using walletStore here?

Reply:
We got a requirement that only share NFT button will be shown to the owner not other users, so for implementing it we have to access walletStore there.

Reply:
Then isnt it should be added in the viewModel?